observatorium/base/observatorium.yaml

```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: core.observatorium.io/v1alpha1
+kind: Observatorium
+metadata:
+  labels:
+    app.kubernetes.io/instance: opf-observatorium
+    app.kubernetes.io/name: observatorium-cr
+    app.kubernetes.io/part-of: observatorium
+  name: opf-observatorium
+spec:
+  # Only Loki Components are scaled up
+  loki:
+    image: quay.io/app-sre/loki:1.6.1
+    replicas:
+      distributor: 1
+      ingester: 1
+      querier: 1
+      query_frontend: 1
+      table_manager: 1
+    version: 1.6.1
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 250Mi
+  objectStorageConfig:
+    loki:
+      accessKeyIdKey: aws_access_key_id
+      bucketsKey: buckets
+      endpointKey: endpoint
+      regionKey: region
+      secretAccessKeyKey: aws_secret_access_key
+      secretName: loki-objectstorage
```

Comment:
Can we have a "stub", "dummy" secret as part of the base manifests? With empty values for example, maybe excluded from the `kustomization.yaml` and so on. So we just know what values we have to fill in to make it work, and we don't have to dig into different manifests...

Reply:
good idea :+1:

Reply:
Added an example secret for loki (https://github.com/operate-first/apps/blob/4feded3993df466023a62c232c7f455831efcff6/observatorium/base/example-loki-objectstorage-secret.yaml)

Reply:
can you please list the rest of the keys in there as well? The `aws_secret_access_key` `aws_access_key_id` etc? So the picture is complete?

Reply:
These keys are supposed to work according to the observatorium documentation, but only the `endpoint` works for now.
I have an issue open upstream for it. (https://github.com/observatorium/operator/issues/31)

Reply:
let's replace them with stub values in the manifest and link to the issue then:

```
objectStorageConfig:
    loki:
      # FIXME: Most of the following values are not used, yet they are required to be defined 
      # See: https://github.com/observatorium/operator/issues/31
      accessKeyIdKey: ""
      bucketsKey: ""
      endpointKey: endpoint
      regionKey: ""
      secretAccessKeyKey: ""
      secretName: loki-objectstorage
```

or something along those lines. So we don't have to wonder where the values come from and the awareness of such problem is shared and tracked. :wink:

Reply:
updated :+1: